src/videojs-http-streaming.js

```diff
@@ -1062,11 +1062,34 @@ class VhsHandler extends Component {
 
     this.player_.tech_.on('keystatuschange', (e) => {
       if (e.status === 'output-restricted') {
-        this.masterPlaylistController_.blacklistCurrentPlaylist({
-          playlist: this.masterPlaylistController_.media(),
-          message: `DRM keystatus changed to ${e.status}. Playlist will fail to play. Check for HDCP content.`,
-          blacklistDuration: Infinity
-        });
+        const masterPlaylist = this.masterPlaylistController_.media();
```

Comment:
`this.masterPlaylistController_.media()` actually returns the currently active media playlist. You want https://github.com/videojs/http-streaming/blob/97e02fb73e227d9f82c0594f42f0575a8f89a29c/src/master-playlist-controller.js#L1665

```suggestion
        const masterPlaylist = this.masterPlaylistController_.master();
```